docs:#372 add links (#1227)
redpanda-data/documentation: modules/reference/pages/internal-metrics-reference.adoc

----
====

> Internal metrics (`/metrics`) can generate thousands of metric series in production environments. Use them judiciously in monitoring systems to avoid performance issues. For alerting and dashboards, prefer public metrics (`/public_metrics`) which are optimized for lower cardinality.

Reviewer: ```suggestion
Internal metrics (`/metrics`) can generate thousands of metric series in production environments, so use them judiciously in monitoring systems to avoid performance issues. For alerting and dashboards, public metrics are preferable (`/public_metrics`) because they are optimized for lower cardinality.
```
